### PR TITLE
Merge testcontainers and fix issues in hadoop starting with branch 0.4

### LINKED ISSUE #1734: Branch 0.4 and master are failing on travis ci
Hadoop tests are failing.

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #1734

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -108,33 +109,11 @@ matrix:
...
     - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cql' CASSANDRA_VERSION='3.0.14' ARGS='-Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-ssl=true'
+      env: MODULE='cql' ARGS=' -Pcassandra3-murmur -Dcassandra.docker.version=3.0.18'
     - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cql' CASSANDRA_VERSION='3.0.14' CASSANDRA_ENABLE_SSL='true' ARGS='-Dtest.skip.murmur=true -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true'
-    - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='hadoop-parent/janusgraph-hadoop-2' CASSANDRA_VERSION='2.2.10' ARGS='-DskipHBase'
-    - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cassandra' CASSANDRA_VERSION='2.2.10' CASSANDRA_ENABLE_BOP='true' ARGS='-Dtest=**/thrift/* -Dtest.skip.unordered=true -Dtest.skip.ssl=true -Dtest.skip.serial=true'
-    - if: type = cron OR commit_message =~ /\[full build\]/
```
Comment (porunov) [inline .travis.yml]: Why you removed all cassandra 2 tests from `[full build]?`

Comment (farodin91) [reply .travis.yml]: They are duplicated. The default version of Cassandra is in branch 2.2. So they are tested already with the default tests of cassandra and cql.

**Comment by farodin91** [comment]
I would prefer to disable to old bot.

**Comment by farodin91** [comment]
Master branch did pass: https://travis-ci.com/farodin91/janusgraph/builds/123898430

**Pull request merged**